### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by janhavisinghh**

### CODE REVIEW

**Review by msasikanth — CHANGES_REQUESTED** [review]

**Review by msasikanth — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 ## Next Release
 ### Internal
+- Bump Kotlin to v1.5.10
+- Bump AGP to v4.2.1
+
+## Features
+- Add NHID support in `PatientEntryScreen`
```
Comment (msasikanth) [inline CHANGELOG.md:9]: Shouldn't this be [in-progress] considering you're still working on it?

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (msasikanth) [reply CHANGELOG.md:9]: or is it a different flow?

Comment (janhavisinghh) [reply CHANGELOG.md:9]: This is Honey's changes of adding support to show nhid. Mine is prefilling.

**Review by msasikanth — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (janhavisinghh) [reply CHANGELOG.md:9]: https://github.com/simpledotorg/simple-android/pull/2652

**Pull request merged**